framework/resources/Advanced/runtimefilter/tpch_sf100/parquet/basic_broadcast.q

```diff
@@ -0,0 +1,16 @@
+alter session set `exec.hashjoin.enable.runtime_filter` = true;
```

Comment:
```suggestion
set `exec.hashjoin.enable.runtime_filter` = true;
```
This is the convention we've been following - 'alter session' is not needed - the recommended is the shorthand 'set', which has session scope by default. Also, only session options are permitted within tests, so there cannot be any confusion with 'alter system'.

Reply:
Okay. Done.